src/cache.js

```diff
@@ -0,0 +1,29 @@
+var _cache = [];
+
+function EventHandler(token, eventName, handler, enabled) {
+  this.elementToken = token;
+  this.eventName = 'click.namespacename';
+  this.eventType = 'click';
+  this.namespace = 'namespace';
+  this.handler = handler;
+  this.enabled = enabled;
+}
+
+module.exports = {
+  add : function(token, eventName, handler) {
+    _cache.push(new EventHandler(token, eventName, handler, true));
+  },
+  removeHandler : function(token, handler) {
+    var index = _cache.findIndex(function(entry) {
+      return entry.elementToken === token && entry.handler === handler;
+    });
+    _cache.splice(index, 1);
+  },
```

Comment:
I suppose it better have. I guess the same handler can be used for the same element but under a different eventName. I'll add this check.